content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -0,0 +1,176 @@
+<!-- TODO: mention preview feature + version info in intro -->
+
+## Handle relations in your relational database with the <inlineCode>foreignKeys</inlineCode> relation mode
+
+The `foreignKeys` relation mode handles relations in your relational database using foreign keys. This is the default option when you use a relational database connector (PostgreSQL, MySQL, SQLite, SQL Server, CockroachDB).
+
+The `foreignKeys` relation mode is not available when you use the MongoDB connector. Some relational databases, [such as PlanetScale](/guides/database/using-prisma-with-planetscale#how-to-emulate-referential-integrity-in-prisma-client), also forbid the use of foreign keys. In these cases, you should instead [emulate relations in Prisma with the `prisma` relation mode](#emulate-relations-in-prisma-with-the-prisma-relation-mode).
+
+### Referential integrity
+
+The `foreignKeys` relation mode enforces referential integrity checks at the database level. This means that when you _create_ or _update_ a record with a relation to another record, the related record needs to exist. If the record does not exist, the database will return an error message.
+
+### Referential actions
+
+When you use the `foreignKeys` relation mode, referential actions are triggered in the database after you _update_ or _delete_ a record with relations to other records. These referential actions maintain referential integrity in the related records by preventing changes that would break referential integrity, propagating changes through to related records or setting the value of fields that reference the updated or deleted record to a null or default value.
+
+For more information, see the [referential actions](/concepts/components/prisma-schema/relations/referential-actions) page.
+
+### Introspection
+
+- When using **Introspection**/`db pull`, on a relational database, `@relation` will be added in the Prisma schema where Foreign Keys exist.
+
+### Prisma Migrate
+
+- When using **Migration** a schema with `@relation`, we turn that into DDL that creates foreign keys
+
+## Emulate relations in Prisma with the <inlineCode>prisma</inlineCode> relation mode
+
+The `prisma` relation mode emulates some referential integrity checks and referential actions for each Prisma Client query, using some additional database queries and logic.
+
+The `prisma` relation mode is the default option for the MongoDB connector. It should also be set if you use a relational database that does not support foreign keys. For example, [if you use PlanetScale](https://www.prisma.io/docs/guides/database/using-prisma-with-planetscale#how-to-enable-emulation-of-referential-integrity) you should use the `prisma` relation mode.
+
+<Admonition type="warning">
+  There are significant performance implications to emulating referential
+  integrity in the Prisma client. In cases where the underlying database can
+  handle referential integrity with foreign keys, it is usually the preferred
+  choice.
+</Admonition>
```

Comment:
See comments on text above.

I like this place for it better though.

Reply:
Added in suggestions from other comment